### ISSUE TITLE
#MindSpore2023 Your Voice Matters — MindSpore New Year's Resolution

### ISSUE THREAD

**Issue by lasdjfi** [mindspore-ai/community#7]
Dear valued developers,

MindSpore is our versatile deep learning framework, and we're constantly looking at new ways to make AI modeling and execution more efficient, and as well as cover more scenarios. With 2023 just around the corner, the MindSpore community would like to wish you a happy holiday – we hope you'll give us some feedback, so that we can create a community that best serves your needs.

We'd appreciate it if you could take a few minutes to answer the three questions below – please write your comments on this page. Thank you for your support!

1.	What changes would you like to see to MindSpore in 2023? Feel free to list a few different suggestions.

2.	Are you interested in MindSpore meetups, courses, tutorials, and conferences? If so, which interests you the most? What do you hope to gain from these activities?

3.	Would you consider using MindSpore in your Google Summer of Code (GSoC) projects in 2023? If yes, please give us more details.

The MindSpore community highly values your feedback. We have prepared special Christmas prizes for a few lucky winners. Leave a comment below for the chance to be picked; all winners will be contacted via GitHub. Merry Christmas and a happy new year!

For more info, follow our social media accounts.
Twitter: https://twitter.com/MindSporeOSS
LinkedIn: https://www.linkedin.com/company/80968760/
YouTube: https://www.youtube.com/@MindsporeOSS
Reddit: https://new.reddit.com/r/MindSporeOSS/

**Comment by LoveKendall** [comment]
More practical and instrumental content that we can use directly

**Comment by Janaki-veligarapu** [comment]
Yeah we are eagerly waiting for new project and very nice post

**Comment by janna225** [comment]
1. Hope MindSpore can enter the metaverse, build its own metacosmic ecology, and create its own avatar
2. Of course. I'd like to see more MindSpore meetups and tutorials, hoping get trends of the industry from professionals, or know their insights about how the platform have done and will go. Also, tutotials are suitable for some newbies like me, who really want to begin an AI journey but are afraid to take the first step.
3. Yes. I'll definately consider that.

**Comment by vashusoni** [comment]
Yes, we are anxiously anticipating a fresh endeavour and an excellent post.

**Comment by Judithsq** [comment]
Tutorials. I think more tutorials are needed to help my future development.

**Comment by LiuPeng-running** [comment]
I'd like to see more webinars in the community. Talks from insiders are always interesting.

**Comment by hcy1012** [comment]
More interesting meetups

**Comment by samipak458** [comment]
1) Tutorials that help in learning(step by step) the framework.
2) For GSSOC, technical and non-technical(Docs) projects should added. Also a sample proposal which will help contributors in understanding what organization is expecting.

**Comment by fidelis1025** [comment]
Waiting for new project and very nice post

**Comment by Yasir761** [comment]
1. There should be tutorials, but with slightly different touch for the framework .
2. I think there should  be virtual meetups more than in person meetups.

**Comment by ListowelAdolwin** [comment]
This is already a great project .
Just a few suggestions
1. There should be tutorials, but with slightly different touch for the framework .
2. I think there should be virtual meetups more than in person meetups

**Comment by Ayobamidele** [comment]
I'm looking forward for more virtual meetups so we have more inclusiveness in the community and tutorials especially for entry level developers.

**Comment by priyapri-23** [comment]
I

**Comment by priyapri-23** [comment]
I would like to attend your meetups and want to learn from your tutorials & conferences to enhance my skill and knowledge, I would like to explore it more & contribute in it , I am new to GSOC so I'll try mindspore also for my GSOC project

**Comment by deadpool620** [comment]
Enhanced support for different hardware platforms: MindSpore currently supports a range of hardware platforms, including CPUs, GPUs, and specialized hardware such as FPGAs and application-specific integrated circuits (ASICs). In the future, it may be useful to expand support for additional hardware platforms in order to enable users to take advantage of the latest hardware innovations.

**Comment by Pranjal360Agarwal** [comment]
• Some interesting techs like DALL-E and Chat-GPT that everyone can play with. this will make the whole project go viral, and more talented coders would want to make a contribution.

• yes, meetups would be a great way to gain hands-on experience and learn from experts in the field.

• Sure it will add the best worthy value for GsoC

**Comment by aitianshi** [comment]
### What changes would you like to see to MindSpore in 2023?

1. Enhanced usability features such as an improved graphical user interface to simplify complex operations.
2. Improved support for distributed computing, allowing for faster and more efficient training of large datasets.
3. Enhanced support for efficient and automated model deployment.
4. Improved support for edge computing, allowing for more efficient deployment of models on low-power devices.

### Are you interested in MindSpore meetups, courses, tutorials, and conferences? 

I am very interested. Of these activities, I find the courses and tutorials the most interesting, as they provide the opportunity to gain a deep understanding of the MindSpore platform. I hope to gain a comprehensive understanding of how to use MindSpore to develop AI applications and use it effectively.
